open-metadata-implementation/access-services/asset-catalog/asset-catalog-api/src/main/java/org/odpi/openmetadata/accessservices/assetcatalog/exception/AssetCatalogErrorCode.java

```diff
@@ -49,12 +49,17 @@
     LINKING_RELATIONSHIPS_NOT_FOUND(404, "OMAS-ASSET-CATALOG-404-015 ",
             "There is no intermediate relationships that connect the {0} with the {1} in OMAS Server {2}",
             "The system is unable to retrieve the linking relationship.",
-            "Check that the unique identifiers of the assets are correct."),
+            Constants.CORRECT_UNIQUE_IDENTIFIER),
```

Comment:
I do no think we should use constants in our messages. These messages should be localisable and a constant makes that very difficult to do.